src/Client/Web/Bit.Client.Web.BlazorUI/Styles/Fluent/Notifications/bit-message-bar.fluent.scss

```diff
@@ -1,99 +1,213 @@
 ﻿@import "../colors.fluent.scss", "../../functions.scss";
 
 .bit-msg-bar-fluent {
+    background-color: $Grey20;
+    color: $Grey160;
     font-family: FluentFont;
     font-size: rem(12px);
     font-weight: 400;
-    line-height: inherit;
-    color: $Grey160;
-    align-items: center;
+    min-height: 32px;
     width: 100%;
     display: flex;
     word-break: break-word;
-    min-height: 32px;
-    align-items: flex-start;
 
-    & > span:nth-child(2) {
-        flex-grow: 1;
-        margin: 8px 0px 8px 12px;
-        padding: 0;
-        text-align: left;
-        overflow: hidden;
-        white-space: nowrap;
-        text-overflow: ellipsis;
+    &.bit-msg-bar--error, &.bit-msg-bar--blocked {
```

Comment:
&.bit-msg-bar--error will be generated to .bit-msg-bar-fluent.bit-msg-bar--error
selecting a tag with two of its classes makes a powerful selector which would prevent consumers to easily override our styles please change them to two separate selectors please check the rest of styles for these selectors